subsys/input/Kconfig

```diff
@@ -86,4 +86,32 @@ config INPUT_LONGPRESS
 	default y
 	depends on DT_HAS_ZEPHYR_INPUT_LONGPRESS_ENABLED
 
+config INPUT_LVGL_POINTER
+	bool "Input lvgl pointer"
+	default y
+	select INPUT_LVGL
+	depends on LVGL
+	depends on DT_HAS_ZEPHYR_INPUT_LVGL_POINTER_ENABLED
+
+if INPUT_LVGL_POINTER
+
+config INPUT_LVGL_POINTER_MSGQ_COUNT
+	int "Message queue size"
+	default 10
+	help
+	  Size of the pointer message queue buffering input events.
```

Comment:
how about having a single config for the queue size for all input events? keep the config a bit simpler

Reply:
Thought about that one for a bit, I wanted to have this per `indev` type because, a touch controller with interrupt generated `input_events` will want to have a bigger queue then a button, that you can most likely only press once within the lvgl refresh period. Tried saving some flash :^)

Reply:
Sounds good.